src/main/java/org/apache/sling/jcr/repoinit/impl/AclUtil.java

```diff
@@ -420,7 +419,7 @@ private static boolean isValidPath(@NotNull Session session, @Nullable String jc
     /**
      * 
      * @param acMgr the access control manager
-     * @param principal the principal
+     * @param path the path
```

Comment:
is that change related?

Reply:
Nope, just javadoc cleanup. I'll revert it.

Reply:
don;t get me wrong..... i am all for cleaning up code and actively maintaining it... but i would love this to happen with a separate PR. such that we can really track what has changed and why.